### PR TITLE
added router

### PR DESCRIPTION
**Pull request by Rebal67**
issue #84

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
-import TButton from './src/components/Tbutton';
+import { User, JsKeyManager } from 'tonomy-id-sdk';
+import { useEffect } from 'react';
+import MainNavigation from './src/navigation/main';
+
+
+
 export default function App() {
+
+  useEffect(() => {
```
Comment (theblockstalk) [inline App.tsx:10]: Unused function, remove

File: src/screens/testScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import TButton from '../components/Tbutton';
+import { StyleSheet, Text, View } from 'react-native';
+import { NavigationProp } from '@react-navigation/native';
+export default function HomeScreen({ navigation }: { navigation: NavigationProp<any> }) {
```
Comment (theblockstalk) [inline src/screens/testScreen.tsx:4]: add space before, remove extra spaces L5-6

File: App.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rebal67) [reply App.tsx]: done

File: src/screens/testScreen.tsx (hunk 2 above)
Comment (Rebal67) [reply src/screens/testScreen.tsx]: done

**Review by theblockstalk — APPROVED** [review]

**Pull request merged**